src/app/admin/_components/Specialist/Create.js

```diff
@@ -39,7 +39,7 @@ export function SpecialistCreate() {
         >
```

Comment:
Could you please remove two lines 36 and 37 which are now commented out? I forgot to delete it while working on the specialist creation form.

Reply:
done